ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
                              ^^^^^^^^
NameError: name 'BAUDRATE' is not defined. Did you mean: 'AUDRATE'?

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
                              ^^^^^^^^
NameError: name 'BAUDRATE' is not defined. Did you mean: 'AUDRATE'?

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
                              ^^^^^^^^
NameError: name 'BAUDRATE' is not defined. Did you mean: 'AUDRATE'?

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
                              ^^^^^^^^
NameError: name 'BAUDRATE' is not defined. Did you mean: 'AUDRATE'?

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialwin32.py", line 33, in __init__
    super(Serial, self).__init__(*args, **kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialutil.py", line 244, in __init__
    self.open()
    ~~~~~~~~~^^
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialwin32.py", line 64, in open
    raise SerialException("could not open port {!r}: {!r}".format(self.portstr, ctypes.WinError()))
serial.serialutil.SerialException: could not open port '/dev/ttyACM0': FileNotFoundError(2, 'ÁöÁ¤µÈ °æ·Î¸¦ Ã£À» ¼ö ¾ø½À´Ï´Ù.', None, 3)

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
ERROR:root:Traceback (most recent call last):
  File "c:\Users\a\Documents\GitHub\rpi-edb\lib.py", line 12, in read
    ser = serial.Serial(PORT, BAUDRATE, timeout=2)  # timeout Áõ°¡
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialwin32.py", line 33, in __init__
    super(Serial, self).__init__(*args, **kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialutil.py", line 244, in __init__
    self.open()
    ~~~~~~~~~^^
  File "C:\Users\a\Documents\GitHub\rpi-edb\.venv\Lib\site-packages\serial\serialwin32.py", line 64, in open
    raise SerialException("could not open port {!r}: {!r}".format(self.portstr, ctypes.WinError()))
serial.serialutil.SerialException: could not open port '/dev/ttyACM0': FileNotFoundError(2, 'ÁöÁ¤µÈ °æ·Î¸¦ Ã£À» ¼ö ¾ø½À´Ï´Ù.', None, 3)

INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.0.31:5001
INFO:werkzeug:[33mPress CTRL+C to quit[0m
